homeassistant/components/squeezebox/media_player.py

```diff
@@ -284,6 +321,24 @@ def supported_features(self):
         """Flag media player features that are supported."""
         return SUPPORT_SQUEEZEBOX
 
+    @property
+    def sync_group(self):
+        """
+        List of players we are synced with.
+
+        If the player is a squeezebox entity, returns the entity id. If it is unknown,
+        return the player_id.
```

Comment:
Isn't it confusing to return different ids from the same property?

Reply:
I suppose it is. I was worried about silently failing on squeezeboxes that are synced but for some reason not tracked in homeassistant. We can't do anything useful with squeezeboxes that are not entities, so I will log an error instead.